roles/splunk/tasks/configure_facl.yml

```diff
@@ -41,4 +46,6 @@
         - restart non-redhat auditd service
       when: result_auditd_conf.stat.exists
 
-  when: splunk_nix_user != 'root'
+  when:
+    - least_privileged is not defined or "'full' in group_names"
```

Comment:
As of version 9.0.0.x, least_privileged mode is not yet available on full installation. This may change in the future